Error installing node-zwave-js 10.0.1

```
❯ npm install zwave-js@10.0.1 @zwave-js/server@1.22.0
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: undefined@undefined
npm ERR! Found: zwave-js@10.0.1
npm ERR! node_modules/zwave-js
npm ERR!   zwave-js@"10.0.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer zwave-js@"10.0.0" from @zwave-js/server@1.22.0
npm ERR! node_modules/@zwave-js/server
npm ERR!   @zwave-js/server@"1.22.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

Looks like the dependency version is too strict.
https://github.com/zwave-js/zwave-js-server/blob/1973f99dc0c243389dc311152b5f55da2d2573d1/package.json#L37

--- a/package.json
+++ b/package.json
@@ -34,7 +34,7 @@
     "@homebridge/ciao": "^1.1.3"
   },
   "peerDependencies": {
-    "zwave-js": "^10.0.0"
+    "zwave-js": "^10.0.1"
   },
   "devDependencies": {
     "@types/minimist": "^1.2.1",
@@ -53,7 +53,7 @@
     "semver": "^7.3.5",
     "ts-node": "^10.5.0",
     "typescript": "^4.1.3",
-    "zwave-js": "^10.0.0"
+    "zwave-js": "^10.0.1"
   },
   "husky": {
     "hooks": {

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@zwave-js/server",
-  "version": "1.22.0",
+  "version": "1.22.1",
   "description": "Full access to zwave-js driver through Websockets",
   "homepage": "https://github.com/zwave-js/zwave-js-server#readme",
   "repository": {
@@ -34,7 +34,7 @@
     "@homebridge/ciao": "^1.1.3"
   },
   "peerDependencies": {
-    "zwave-js": "10.0.0"
+    "zwave-js": "^10.0.0"
   },
   "devDependencies": {
     "@types/minimist": "^1.2.1",
@@ -53,7 +53,7 @@
     "semver": "^7.3.5",
     "ts-node": "^10.5.0",
     "typescript": "^4.1.3",
-    "zwave-js": "10.0.0"
+    "zwave-js": "^10.0.0"
   },
   "husky": {
     "hooks": {

